portfolio/src/test/java/com/google/sps/CollectEventsFromJsonArrayTest.java

```diff
@@ -0,0 +1,56 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Collection;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class CollectEventsFromJsonArrayTest {
```

Comment:
I think this file should be named ServletHelperTest. The test file needs to be associated with the Java file.

Reply:
Done.